src/src/tx_main.cpp

```diff
@@ -513,14 +528,9 @@ void sendELRSstatus()
   }
 
 void resetLuaParams(){
-  setLuaTextSelectionValue(&luaAirRate,(uint8_t)(ExpressLRS_currAirRate_Modparams->index));
-  setLuaTextSelectionValue(&luaTlmRate,(uint8_t)(ExpressLRS_currAirRate_Modparams->TLMinterval));
-  
-  #ifdef USE_DYNAMIC_POWER
-  setLuaTextSelectionValue(&luaPower,(uint8_t)(config.GetPower()));
-  #else
-  setLuaTextSelectionValue(&luaPower,(uint8_t)(POWERMGNT.currPower()));//value
-  #endif
+  setLuaTextSelectionValue(&luaAirRate, (uint8_t)config.GetRate());
+  setLuaTextSelectionValue(&luaTlmRate, (uint8_t)config.GetTlm());
+  setLuaTextSelectionValue(&luaPower, (uint8_t)config.GetPower());
```

Comment:
Why this last change? Do we not want the currentPower level if using dynamic power?
Actually I think this is better, but we should perhaps add an info field with the current power level as part of the LUA changes perhaps. Or undo this section from this commit and add it to the LUA change patch?

Reply:
No, the configuration system is for changing the configuration, not getting the current status. The current status is available from the Telemetry system. If the user is changing a setting, then they need to see what it is currently set at, not the current status which misleads them to thinking they're *configured* to that value. If it displays the current value, then how does the user know what it is set to?

If you make this change in the Lua PR that's fine too, I'll fix up the conflict but it needs to be done somewhere because this is the only way the user can see their configured setting.